Source/WebKit/UIProcess/API/Cocoa/WKWebExtensionContext.mm

```diff
@@ -875,6 +881,11 @@ - (BOOL)isLoaded
     return NO;
 }
 
+-(NSArray<NSError *> *)errors
+{
+    return nil;
```

Comment:
This property is not marked `nullable`, and the header documentation specifically says an empty array is returned when no errors are available.
```suggestion
    return @[ ];
```

Reply:
Just be aware that if/when we enable the static analyzer for WebKit this will generate an analyzer warning. The convention we use in internal code is to force cast `(id _Nonnull)nil` to suppress this in cases like this where we intentionally return nil, I'm not sure if we use this in WebKit code.
```suggestion
    return (id _Nonnull)nil;
```